storybook/preview.js

```diff
@@ -155,3 +155,5 @@ export const parameters = {
 	},
 	sourceLinkPrefix: 'https://github.com/WordPress/gutenberg/blob/trunk/',
 };
+
+export const tags = [ 'autodocs' ];
```

Comment:
The [migrations docs](https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#mainjs-docsautodocs-is-deprecated) suggesting adding a default export, is there a reason why we're not following that suggestion and opting for a named export instead? Could this ever stop working in the future?

Reply:
The auto migrator didn't have a problem catching that and working with it. I think it shouldn't be an issue. If it does, we'll find out quickly enough.